### PR TITLE
Adding new repository for indexing: md25_driver

### PR DESCRIPTION
**Pull request by ozoid**
MD25 Motor Driver with Odometry tf Twist PID

### CODE REVIEW

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -5281,6 +5281,16 @@ repositories:
       url: https://github.com/at-wat/mcl_3dl_msgs.git
       version: master
     status: developed
+  md25_driver:
+    doc:
+      type: git
+      url: https://github.com/ozoid/md25_driver.git
```
Comment (clalancette) [inline melodic/distribution.yaml:5287]: The `package.xml` in your source repository mentions a license of `Apache 2.0`, which is good.  However, you have no LICENSE file in your repository.  Can you please add one?

**Review by clalancette — CHANGES_REQUESTED** [review]

File: melodic/distribution.yaml (hunk 1 above)
Comment (ozoid) [reply melodic/distribution.yaml:5287]: Hi,
Licence changed to MIT - and licence file added as requested, sorry for the issue.
Thank you

Comment (clalancette) [reply melodic/distribution.yaml:5287]: So the problem with changing the license is that this is a forked repository.  So you can't unilaterally change the license without agreement from the original author (@allenh1 in this case).  Your options are:

1.  Get agreement from @allenh1 to switch the license to MIT.
1.  Switch the license back to Apache 2.0 like the original, and add a LICENSE file saying that.

Comment (allenh1) [reply melodic/distribution.yaml:5287]: @clalancette thanks for the ping.

@ozoid have you significantly progressed the package to a useful state? I had only started working on this when I blew out my motor board by connecting the battery wires to the wrong terminals (they really should put labels on the battery connectors).

I'm glad to discuss switching to MIT, but let's do that in a pull request from your repo into mine. If you want to progress it forward, we can consider transferring the repository to you or giving you privileges on mine.

Cheers.

**Comment by clalancette** [comment]
I'm going to close this PR out for now until the licensing situation is sorted out.  Once that is done, please feel free to open a new pull request.

File: melodic/distribution.yaml (hunk 1 above)
Comment (ozoid) [reply melodic/distribution.yaml:5287]: I am using the new code - and it seems to be working well on my robot.
I was unsure of what licence to have/choose and didn't know there would be complications. I am happy to go with the flow, as long as others get to use the code too :-) I wanted to be included in the ROS repo so I/the code could be famous ;-) I couldn't find something suitable that was already developed.

Comment (allenh1) [reply melodic/distribution.yaml:5287]: > I am using the new code - and it seems to be working well on my robot.

Overall looks like some very good, solid contributions to a very unfinished package.

> I was unsure of what licence to have/choose and didn't know there would be complications.

No worries, I'm just glad @clalancette caught it.

>  am happy to go with the flow, as long as others get to use the code too :-)

I'm very glad to hear this! Let's work on merging your fork into my repo, then we can make you a maintainer.

>  I couldn't find something suitable that was already developed.

Yep -- apologies for that. Finishing the code has been on my to-do list for at least a few years now.

Cheers!

**Pull request closed without merging**